tools/HelperUtilities/psScripts/Copy-SessionRecords.ps1

```diff
@@ -0,0 +1,48 @@
+# ---------------------------------------------------------------------------------- 
+    # Copyright (c) Microsoft Corporation. All rights reserved.
+    # Licensed under the MIT License. See License.txt in the project root for
+    # license information.
+# ---------------------------------------------------------------------------------- 
+
+<#
+
+.SYNOPSIS
+    Powershell script that copies newly recorded session records from source to destination
+
+.DESCRIPTION
+    This script copies newly recorded session records from source to destination
+
+.PARAMETER Source
+    Full path to directory where session records are (usually the bin directory of recorded test)
+
+.PARAMETER Destination
+    Location where to copy the files
+#>
+
+Param(
+    [Parameter(Mandatory = $true)]
+    [string] $Source,
```

Comment:
@dsgouda make the parameter as SourceRootDirectory and DestinationRootDirectory.
Also add a help message attribute that will allow people to get some basic help as to what is that the user wants to provide.
Also add -whatIf, because that will allow users to know what will get copied or changed without really executing your script.
So the best way to implement -whatIf is to implement SupportShouldProcess, this will provide -whatIf and -Confirm to the user.